lib/common/socket.c

```diff
@@ -1470,3 +1470,112 @@ void h2o_sliding_counter_stop(h2o_sliding_counter_t *counter, uint64_t now)
     /* recalc average */
     counter->average = counter->prev.sum / (sizeof(counter->prev.slots) / sizeof(counter->prev.slots[0]));
 }
+
+#if H2O_USE_DTRACE && defined(__linux__)
+#include <linux/bpf.h>
+#include <linux/unistd.h>
+#include "h2o-probes.h"
+#include "include/h2o/ebpf.h"
+#include <sys/stat.h>
+
+int h2o_tracing_map_fd = 0;
+uint32_t h2o_tracing_map_last_attempt = 0;
```

Comment:
I might suggest converting this variable to a static variable of the open_map function assuming that the variable is only used from that function.